invenio/modules/editor/bundles.py

```diff
@@ -0,0 +1,48 @@
+# -*- coding: utf-8 -*-
+##
+## This file is part of Invenio.
+## Copyright (C) 2015 CERN.
+##
+## Invenio is free software; you can redistribute it and/or
+## modify it under the terms of the GNU General Public License as
+## published by the Free Software Foundation; either version 2 of the
+## License, or (at your option) any later version.
+##
+## Invenio is distributed in the hope that it will be useful, but
+## WITHOUT ANY WARRANTY; without even the implied warranty of
+## MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+## General Public License for more details.
+##
+## You should have received a copy of the GNU General Public License
+## along with Invenio; if not, write to the Free Software Foundation, Inc.,
+## 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+
+"""Editor bundles."""
+
+from invenio.ext.assets import Bundle
+
+js = Bundle(
+    "vendors/jquery/dist/jquery.js",
```

Comment:
maybe you want to use `invenio.base.bundles.jquery` instead. See following example https://github.com/inveniosoftware/invenio/blob/pu/invenio/modules/deposit/bundles.py#L25-L38